lib/Conversion/TritonGPUToLLVM/ConvertLayoutOpToLLVM/SharedToDotOperandMMAv2.cpp

```diff
@@ -106,25 +106,28 @@ MMA16816SmemLoader::computeLdmatrixMatOffs(Value warpId, Value lane,
   Value kMatArr = kOrder == 1 ? s1 : s0;
   Value nkMatArr = kOrder == 1 ? s0 : s1;
 
-  // matrix coordinate inside a CTA, the matrix layout is [2x2wpt] for A and
-  // [2wptx2] for B. e.g. Setting wpt=3, The data layout for A(kOrder=1) is
-  //   |0 0 1 1 2 2| -> 0,1,2 are the warpids
-  //   |0 0 1 1 2 2|
-  //
-  // for B(kOrder=0) is
-  //   |0 0|  -> 0,1,2 are the warpids
-  //   |1 1|
-  //   |2 2|
+  // Matrix coordinates inside a CTA,
+  // the matrix layout is [2x2wpt] for A and [2wptx2] for B.
+  // e.g., Setting wpt=4, the data layout for A(kOrder=1) is
+  //   |0 0|  -> 0,1,2,3 are the warpids
```

Comment:
Not sure if I understand it correctly, I think A and B's layout were reversed.

Reply:
Yeah, I think you're right, although I believe it's more something like `[2wpt[0] x 2] for A and [2 x 2wpt[1]] for B`, where wpt is the getParent().getLayout().cast<MMAOperand>().getWarpsPerTile().

Reply:
Thanks! Just fixed the comments based on your suggestion.